data/korath/korath culture conversations.txt

```diff
@@ -0,0 +1,76 @@
+			`	Reaching in her garments, she pulls out what looks to be some form of communication device. She considers it briefly, then gestures for you to follow her back to the shrine, where she carefully places the object down onto the pile of offerings. She then looks at you, as if wondering if you would follow the gesture.`
+			choice
+				`	(Offer 100 credits.)`
+					to display
+						"credits" >= 100
+					goto "small credits"
+				`	(Offer 10,000 credits.)`
+					to display
+						"credits" >= 10000
+					goto "medium credits"
+				`	(Offer 10,000,000 credits.)`
+					to display
+						"credits" >= 10000000
+					goto "large credits"
+				`	(Offer a picture of your home planet.)`
+				`	"I'm sorry, but I have nothing to offer. Goodbye."`
+					goto end
+			`	You remember the picture of your hometown back in <starting planet> you tend to carry around, and start searching for it. Sure enough, you find it inside your back pocket, straighten it out as best as you can, and go to add it to the pile. The Korath stops you, however, holding your arm, and stretching her hand carefully, as if asking for the picture. You hand it to her, and she brings it very close to her eyes. The elderly Korath who wailed before appears beside her, and she shows him the picture. He seems to stare at it for minutes on end, and you begin to wonder if you somehow offended them with your choice. Then, he hands her back the picture, and seems to nod slightly. She heads close to the shrine, and places the picture at the top level, with the feather, shell, and flower. The elderly Korath faces you, looking deeply into your eyes, then raises his palms toward you, letting out a faint bark. You notice all other Korath around the shrine are doing the same. When they've all acknowledged your contribution, the Korath who drew the spiral arms comes to you again, looking thoroughly impressed.`
+				goto end
+			label "small credits"
+			action
+				payment -100
+			`	You add the credit chip to the objects by the shrine, and she seems to nod approvingly, before looking at you once more, seemingly satisfied with your understanding of the gesture.`
+				goto end
+			label "medium credits"
+			action
+				payment -10000
+			`	You add the credit chips to the objects by the shrine, and she and a few of the other Korath seem to bow lightly to you. She turns to you once more, looking satisfied with your understanding of the gesture.`
+				goto end
+			label "large credits"
+			action
+				payment -10000000
+			`	You attempt to carefully place the credit chips onto the pile of objects by the shrine, but despite your efforts a few end up slipping out and dropping down rather noisily. The Korath, however, don't seem all disturbed, but rather in shock at your addition. Many of them raise their hands up, their palms to you, in greeting, or as if thanking you for your contribution. You look back at the Korath who drew the spiral arms, who seems to be beaming.`
```

Comment:
```suggestion
			`	You attempt to carefully place the credit chips onto the pile of objects by the shrine, but despite your efforts a few end up slipping out and dropping down rather noisily. The Korath, however, seem in shock rather than disturbed at your addition. Many of them raise their hands up, their palms to you in greeting or in thanks. You look back at the Korath who drew the spiral arms, who seems to be beaming.`
```